src/main/resources/org/jboss/xavier/analytics/rules/SourceCosts.drl

```diff
@@ -2,24 +2,118 @@ package org.jboss.xavier.analytics.rules;
+        sourceCostsModel.setYear2SourceValue((double)round(2 * pricing.getSourceListValue() * (1 - pricing.getSourceDiscountPercentage()) * (sourceCostsModel.getYear2Server() - sourceCostsModel.getYear1Server())));
+        sourceCostsModel.setYear3SourceValue((double)round(2 * pricing.getSourceListValue() * (1 - pricing.getSourceDiscountPercentage()) * (sourceCostsModel.getYear3Server() - sourceCostsModel.getYear2Server())));
+        sourceCostsModel.setTotSourceValue(sourceCostsModel.getYear1SourceValue() + sourceCostsModel.getYear2SourceValue() + sourceCostsModel.getYear3SourceValue());
+
+        sourceCostsModel.setYear1SourceMaintenanceValue((double)round(pricing.getSourceMaintenancePercentage() * pricing.getSourceListValue() * sourceCostsModel.getYear1Server()));
+        sourceCostsModel.setYear2SourceMaintenanceValue((double)round(pricing.getSourceMaintenancePercentage() * pricing.getSourceListValue() * sourceCostsModel.getYear2Server()));
+        sourceCostsModel.setYear3SourceMaintenanceValue((double)round(pricing.getSourceMaintenancePercentage() * pricing.getSourceListValue() * sourceCostsModel.getYear3Server()));
+        sourceCostsModel.setTotSourceMaintenanceValue(sourceCostsModel.getYear1SourceMaintenanceValue() + sourceCostsModel.getYear2SourceMaintenanceValue() + sourceCostsModel.getYear3SourceMaintenanceValue());
+
+        sourceCostsModel.setTotalSourceValue(sourceCostsModel.getTotSourceValue() + sourceCostsModel.getTotSourceMaintenanceValue());
+
+        sourceCostsModel.setSourceNewHighValue((double)round((1 + pricing.getSourceRenewHighFactor()) * 3 * 2 *  sourceCostsModel.getSourceMaintenanceValue() * environmentModel.getHypervisors()));
+        sourceCostsModel.setSourceNewLikelyValue((double)round((1 + pricing.getSourceRenewLikelyFactor()) * 3 * 2 *  sourceCostsModel.getSourceMaintenanceValue() * environmentModel.getHypervisors()));
+        sourceCostsModel.setSourceNewLowValue((double)round((1 + pricing.getSourceRenewLowFactor()) * 3 * 2 *  sourceCostsModel.getSourceMaintenanceValue() * environmentModel.getHypervisors()));
+        modify(report)
+        {
+            setSourceCostsModel(sourceCostsModel);
+        }
+end
+
+rule "SourceCostsRules_sourceNewELAIndicator_2"
+    when
+        report : InitialSavingsEstimationReportModel(
+            sourceCostsModel != null,
+            sourceCostsModel.totalSourceValue != null,
+            sourceCostsModel.sourceNewHighValue != null,
+            sourceCostsModel.sourceNewLikelyValue != null,
+            sourceCostsModel.sourceNewLowValue != null,
+            sourceCostsModel.totalSourceValue > sourceCostsModel.sourceNewHighValue
+        )
+    then
+        modify(report)
+        {
+            getSourceCostsModel().setSourceNewELAIndicator(2),
+            getSourceCostsModel().setSourceRenewHighValue(report.getSourceCostsModel().getTotalSourceValue()),
```

Comment:
Are the 3 values set on lines 73-75 meant to be identical?

Reply:
yep, as discussed they have to be the same when `sourceNewELAIndicator` is 2